web/src/main/java/org/openmrs/web/filter/StartupFilter.java

```diff
@@ -177,9 +178,9 @@ private void initializeVelocity() {
 			velocityEngine = new VelocityEngine();
 			
 			Properties props = new Properties();
-			props.setProperty(RuntimeConstants.RUNTIME_LOG, "startup_wizard_vel.log");
+			props.setProperty(RuntimeConstants.RUNTIME_LOG_NAME, "startup_wizard_vel.log");
```

Comment:
Are these changes part of this ticket?

Reply:
@dkayiwa In the update of the dependencies to apache velocity to 2.0., some of these RuntimeConstants like RUNTIME_LOG. are not available [they create an error] in velocity 2.0.